pycufsm/fsm.py

```diff
@@ -413,4 +412,4 @@ def strip(
         # shapes(:,i,1:min([n_modes,num_pos_modes]))=modes
         shapes.append(modes_full)
 
-    return signature.flatten(), curve, shapes
+    return signature.flatten(), curve, shapes
```

Comment:
Re-add a newline at the end of the file here. I'm surprised YAPF didn't give you a warning about this.